pubspec.yaml

```diff
@@ -1,31 +1,31 @@
 name: pana
 description: Evaluate the health and quality of a Dart package
-version: 0.13.5
+version: 0.13.6-dev
 homepage: https://github.com/dart-lang/pana
 
 environment:
-  sdk: '>=2.3.0 <3.0.0'
+  sdk: ">=2.3.0 <3.0.0"
```

Comment:
I didn't notice before pushing, but my formatter replaced all these quotes... I hope this doesn't bother?

Reply:
It would be nice if you could revert those changes, thank you!

Reply:
Also done